cypress/integration/learn/challenges/output.js

```diff
@@ -59,3 +62,24 @@ describe('Classic challenge', function () {
       });
   });
 });
+
+describe('jQuery challenge', function () {
+  before(() => {
+    cy.visit(locations.jQuery);
+  });
+
+  it('renders the default output text', () => {
+    cy.title().should(
+      'eq',
+      'jQuery: Target HTML Elements with Selectors Using jQuery | freeCodeCamp.org'
+    );
+    cy.get(selectors.defaultOutput).contains(defaultOutput);
+  });
+
+  it('should not show a reference error', () => {
+    cy.get(selectors.defaultOutput).should(
+      'not.contain',
+      'ReferenceError: $ is not defined'
```

Comment:
So, the issue with this is that initially this is true, even when the page is broken. My suspicion is that 
```
'contain', 'Your test output will go here'
```
would also work.

What we need to check is that the error never appears.  It's kinda ugly, but we could check that it contains 'Your test output will go here', wait 5 seconds and confirm that the error hasn't appeared.

Reply:
The previous test asserts the `Your test output will go here` - I added a `cy.wait` to this test (which serves mostly as a sanity check just in case)

Reply:
Oh, right. Thanks, Nick, that should do it.